.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pg_major: ["9.6", "10.9", "11.4", "12.2"]
+        variant: [slim]
```

Comment:
It's doubtful that we'll actually add more base image variants but it's the sort of thing I'd expect to see as part of a build matrix. It's also concise to include it here instead of in each include.